components/splitbutton/stories/template.js

```diff
@@ -2,19 +2,66 @@ import { html } from 'lit-html';
 import { classMap } from 'lit-html/directives/class-map.js';
 // import { ifDefined } from 'lit-html/directives/if-defined.js';
 
+import { Template as Button } from '@spectrum-css/button/stories/template.js';
+
 import "../index.css";
 
 export const Template = ({
   rootClass = "spectrum-SplitButton",
   customClasses = [],
   size = "m",
-  // ...globals
+  variant = "cta",
+  iconName = "ChevronDown100",
+  ...globals
 }) => {
+
   return html`
     <div class=${classMap({
       [rootClass]: true,
-      [`${rootClass}--size${size?.toUpperCase()}`]: typeof size !== "undefined",
       ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
-  })}></div>
+    })}>
+      ${Button({
+        ...globals,
+        variant,
+        size: "m",
+        label: "Split Button",
+        customClasses: ["spectrum-SplitButton-action"]
+      })}
+      ${Button({
+        ...globals,
+        variant,
+        size: "m",
+        hideLabel: true,
+        iconName,
+        isOpen: false,
+        customClasses: ["spectrum-SplitButton-trigger"]
+      })}
+    </div>
+    
+    <br />
+    <br />
+    
+    <div class=${classMap({
```

Comment:
I think right & left need to be in the controls so that we can only render 1 split button in the template. That will let us embed the component in others as needed.

Reply:
Picker button & infield button are doing something similar.

Reply:
Oh, yep yep. I wasn't thinking in terms of being able to reuse within other components. Good call-out.